build.gradle

```diff
@@ -60,8 +60,9 @@ allprojects {
     // we allow access to snapshots repo if ALLOW_SNAPSHOT_REPOSITORY is set, what means we are running on CI 
     // with Navigation Native forced to be some snapshot version
     // if you need to use snapshots while development, just set `addSnapshotsRepo` to true manually
-    def addSnapshotsRepo = System.getenv("ALLOW_SNAPSHOT_REPOSITORY") ?: false
+    def addSnapshotsRepo = System.getenv("ALLOW_SNAPSHOT_REPOSITORY")?.toBoolean() ?: false
```

Comment:
These changes shouldn't be included. Rebase issue from https://github.com/mapbox/mapbox-navigation-android/pull/4133? Same thing happens to `dependencies.gradle` changes below.